### ISSUE TITLE
Can not start timescale-prometheus a second time

### ISSUE THREAD

**Issue by mrMoe** [timescale/timescale-prometheus#194]
Hi there,

I am new to timescale db and I am trying to use it as a prometheus long term storage.
So far the db itself is starting fine with docker but the timescale-prometheus bridge/adapter can only be startet successfully once. 
All subsequent executions of the bridge fail with

```
{"caller":"main.go:185","level":"error","msg":"Aborting startup because of migration error: Error while trying to migrate DB: Error encountered during migration: error executing migration script: name idempotent/base.sql, err ERROR: function \"drop_chunks\" already exists with same argument types (SQLSTATE 42723)","ts":"2020-08-16T20:47:14.479Z"}
```

full log

```
root@raspberrypi:/media/pi_data# /media/pi_data/timescale-prometheus/cmd/timescale-prometheus/timescale-prometheus -db-password ***
{"caller":"main.go:161","level":"info","msg":"Version:0.1.0-beta.2.dev.0; Commit Hash: ","ts":"2020-08-16T21:02:53.223Z"}                                                                                                                  
{"caller":"main.go:162","config":"&{listenAddr::9201 telemetryPath:/metrics pgmodelCfg:{host:localhost port:5432 user:postgres password:**** database:timescale sslMode:disable dbConnectRetries:0 AsyncAcks:false ReportInterval:0 LabelsCacheSize:10000 MetricsCacheSize:10000 SeriesCacheSize:0} logLevel:debug haGroupLockID:0 restElection:false prometheusTimeout:-1 electionInterval:5000000000 migrate:true corsOrigin:0x16d8960}","level":"info","ts":"2020-08-16T21:02:53.223Z"}
{"caller":"main.go:173","level":"warn","msg":"No adapter leader election. Group lock id is not set. Possible duplicate write load if running adapter in high-availability mode","ts":"2020-08-16T21:02:53.223Z"}
{"caller":"migrate.go:460","cause":"ERROR: could not open extension control file \"/usr/local/share/postgresql/extension/timescale_prometheus_extra.control\": No such file or directory (SQLSTATE 58P01)","level":"warn","msg":"timescale_prometheus_extra extension not installed","ts":"2020-08-16T21:02:53.621Z"}
{"caller":"client.go:78","level":"info","msg":"host=localhost port=5432 user=postgres dbname=timescale password='****' sslmode=disable connect_timeout=10","ts":"2020-08-16T21:02:53.640Z"}
{"caller":"query_engine.go:13","level":"debug","msg":"Lookback delta is zero, setting to default value","ts":"2020-08-16T21:02:53.658Z","value":"5m0s"}
{"caller":"main.go:278","level":"info","msg":"Starting up...","ts":"2020-08-16T21:02:53.659Z"}
{"addr":":9201","caller":"main.go:279","level":"info","msg":"Listening","ts":"2020-08-16T21:02:53.659Z"}

root@raspberrypi:/media/pi_data# /media/pi_data/timescale-prometheus/cmd/timescale-prometheus/timescale-prometheus -db-password ***
{"caller":"main.go:161","level":"info","msg":"Version:0.1.0-beta.2.dev.0; Commit Hash: ","ts":"2020-08-16T21:05:14.061Z"}
{"caller":"main.go:162","config":"&{listenAddr::9201 telemetryPath:/metrics pgmodelCfg:{host:localhost port:5432 user:postgres password:**** database:timescale sslMode:disable dbConnectRetries:0 AsyncAcks:false ReportInterval:0 LabelsCacheSize:10000 MetricsCacheSize:10000 SeriesCacheSize:0} logLevel:debug haGroupLockID:0 restElection:false prometheusTimeout:-1 electionInterval:5000000000 migrate:true corsOrigin:0x1cb2060}","level":"info","ts":"2020-08-16T21:05:14.062Z"}
{"caller":"main.go:173","level":"warn","msg":"No adapter leader election. Group lock id is not set. Possible duplicate write load if running adapter in high-availability mode","ts":"2020-08-16T21:05:14.062Z"}
{"caller":"main.go:185","level":"error","msg":"Aborting startup because of migration error: Error while trying to migrate DB: Error encountered during migration: error executing migration script: name idempotent/base.sql, err ERROR: function \"drop_chunks\" already exists with same argument types (SQLSTATE 42723)","ts":"2020-08-16T21:05:14.170Z"}
```

**Comment by JLockerman** [comment]
should be fixed by PR #197

**Comment by mrMoe** [comment]
confirm fix.
Thanks for the speedy help!
